README.md

```diff
@@ -1 +1,148 @@
-# React-Capstone
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-react-capstone )
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+  - [🚀 LOOM VIDEO](#Loom-Video)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#Isma)
+- [🔭 Future Features](#the other pages)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#microverse)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [React-Capstone"] <a name="React-Capstone"></a>
+
+**[React-Capstone]** Final Space Wiki is a React-based web application that allows users to explore the world of the animated sci-fi series, Final Space. The app uses the Final Space API to fetch data about the characters, episodes, and locations featured in the show, and presents them in an interactive and user-friendly way.
+
+## 🛠 Built With <a name="HTML, CSS and REACT"></a>
+
+
+### Tech Stack <a name="HTML, CSS and REACT"></a>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/REACT">JavaScript</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Character-Browser]**
+- **[Episode-Guide]**
+- **[Location-Finder]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link] <a href="https://ismailmunyentwari9.github.io/react-capstone/">Go  Live</a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LOOM VIDEO -->
+
+## 🚀 Loom  Video <a name="loom-Video"></a>
```

Comment:
- [ ] Good job with project presentation, however, your missed in your presentation to highlight about a piece of the code (possibly from your code editor) that you find interesting or that you were proud of making. Let's include this missing piece in the video presentation.